### DISCUSSION TITLE
1.6.0 draft

### DISCUSSION THREAD

**Discussion by abarichello** [R2Northstar/Northstar#212] [Draft patch notes]
<div>
    <a><img src="https://user-images.githubusercontent.com/16687318/156849195-06d5c58f-d472-45a7-9ec4-26abc72bc27f.png" width="49%"></a>
    <a><img src="https://user-images.githubusercontent.com/16687318/154815173-454084a4-3b88-48ca-b62b-6df09fe853ff.png" width="49%"></a>
</div>

**Features**
Update private match map select menu ([#194](https://github.com/R2Northstar/NorthstarLauncher/pull/194))
Implemented stub loading which greatly reduces memory usage when running dedicated servers ([#81](https://github.com/R2Northstar/NorthstarLauncher/pull/81))
Added 'Respawn Protection' match setting ([#167](https://github.com/R2Northstar/NorthstarMods/pull/167))
New chathooks available: server broadcasts, whispers and more. ([#74](https://github.com/R2Northstar/NorthstarLauncher/pull/74)) and ([#217](https://github.com/R2Northstar/NorthstarMods/pull/217)). Examples [here](https://r2northstar.readthedocs.io/en/latest/reference/northstar/chathooks.html?highlight=chat#custom-messages)
Playlist overrides now work on listen servers ([#70](https://github.com/R2Northstar/NorthstarLauncher/pull/70))
Updated list of Gun Game weapons ([#252](https://github.com/R2Northstar/NorthstarMods/pull/252))
Auto-generate startup arg files ([#99](https://github.com/R2Northstar/NorthstarLauncher/pull/99))
ConVar class rebuild ([#90](https://github.com/R2Northstar/NorthstarLauncher/pull/90))
Added function to update vortex shield ignored weapons ([#250](https://github.com/R2Northstar/NorthstarMods/pull/250))
Added a default plugin for Discord Rich Presence integration. ([NorthstarDiscordRPC](https://github.com/R2Northstar/NorthstarDiscordRPC))

**Bugfixes**
Update floor is lava riff on maps Crash Site, Drydock, and Complex ([#181](https://github.com/R2Northstar/NorthstarMods/pull/181))
Fixed lethal fog on Deck and Stacks ([#181](https://github.com/R2Northstar/NorthstarMods/pull/181))
Amped Hardpoint now works on Homestead ([#174](https://github.com/R2Northstar/NorthstarMods/pull/174))
Fixed hardpoints getting stuck on amped status ([#199](https://github.com/R2Northstar/NorthstarMods/pull/199))
Implemented amped/unamped hardpoint prop animations
Fixed Last Titan Stading battery spawns on Angel City and Forwardbase Kodai ([#224](https://github.com/R2Northstar/NorthstarMods/pull/224))
Core meter now shows progression towards electric smoke on titans ([#212](https://github.com/R2Northstar/NorthstarMods/pull/212))
Multiple resolutions for the Northstar icon ([#97](https://github.com/R2Northstar/NorthstarLauncher/pull/97))
Updated the localization files

**Backend**
We now have a landing page ([#32](https://github.com/R2Northstar/NorthstarMasterServer/pull/32)), check https://northstar.tf
Added caching of server list ([#30](https://github.com/R2Northstar/NorthstarMasterServer/pull/30))
Added version numbering for master server requests ([#113](https://github.com/R2Northstar/NorthstarLauncher/pull/113))
Add check for player name correctness. ([#86](https://github.com/R2Northstar/NorthstarLauncher/pull/86))
The build pipelines now generate a `md5sum.txt` file, this can be used to ensure that the downloaded release matches with the git repository.

**Contributors:**
@abarichello @GeckoEidechse @pg9182 @BobTheBob9 @emma-miler @barnabwhy @cpdt @KittenPopo @Mauler125 @Dinorush @ASpoonPlaysGames @x3Karma @Orpheus2401 @fvnkhead @PANCHO7532B @JMM889901 @lapaxx @F1F7Y @Neoministein @taskinoz @hummusbird @begin-theadventure @H0L0theBard @xamionex @SamLam140330 @uniboi @laundmo @MysteriousRSA

**Comment by emma-miler** [comment]
https://github.com/R2Northstar/NorthstarLauncher/pull/86
https://github.com/R2Northstar/NorthstarLauncher/pull/126 (all of the exploit stuff)
discord rpc
https://github.com/R2Northstar/NorthstarLauncher/pull/113 version gating
https://github.com/R2Northstar/NorthstarLauncher/pull/74 new squirrel bridge

**Reply by abarichello** [reply]
done

**Reply by abarichello** [reply]
stop 👀 ing me

**Reply by RenanArgent** [reply]
🥴

**Reply by Striketester** [reply]
Update dropping soon (hopefully™)

**Reply by emma-miler** [reply]
you forgor discord rpc @abarichello

**Reply by Striketester** [reply]
lets laugh at the discord peasants

**Reply by Striketester** [reply]
we have big speak on the discord

**Reply by Striketester** [reply]
stay tuned discord folks also can i @ people through this?

**Reply by Striketester** [reply]
im not a coder aswell idk if im even allowed to do this

**Reply by GeckoEidechse** [reply]
I mean, it's just spammy...

**Reply by Striketester** [reply]
then ill stop

**Reply by xamionex** [reply]
;-;
![image](https://user-images.githubusercontent.com/57235791/160503641-846af7b8-d533-489d-897e-6a24dd4973e6.png)
